src/test/codegen/dst-vtable-align-nonzero.rs

```diff
@@ -24,6 +24,7 @@ pub fn eliminates_runtime_check_when_align_1(
     x: &Struct<WrapperWithAlign1<dyn Trait>>
 ) -> &WrapperWithAlign1<dyn Trait> {
     // CHECK: load [[USIZE:i[0-9]+]], {{.+}} !range [[RANGE_META:![0-9]+]]
+    // CHECK-NOT: tail call i64 @llvm.umax.i64
```

Comment:
```suggestion
    // CHECK-NOT: llvm.umax
```
To preserve intent for targets with 32-bit usize.

Reply:
Done.